src/plugins/plugin.legend.js

```diff
@@ -358,12 +353,12 @@ var Legend = Element.extend({
 				// Set the ctx for the box
 				ctx.save();
 
-				ctx.fillStyle = valueOrDefault(legendItem.fillStyle, globalDefault.defaultColor);
+				ctx.fillStyle = valueOrDefault(legendItem.fillStyle, defaultColor);
 				ctx.lineCap = valueOrDefault(legendItem.lineCap, lineDefault.borderCapStyle);
 				ctx.lineDashOffset = valueOrDefault(legendItem.lineDashOffset, lineDefault.borderDashOffset);
 				ctx.lineJoin = valueOrDefault(legendItem.lineJoin, lineDefault.borderJoinStyle);
 				ctx.lineWidth = valueOrDefault(legendItem.lineWidth, lineDefault.borderWidth);
```

Comment:
Not directly related to this PR, but `valueOrDefault(legendItem.lineWidth, lineDefault.borderWidth)` could be cached for minimization.

Reply:
This is done, thanks.